website/docs/docs/cloud/connect-data-platform/about-connections.md

```diff
@@ -101,3 +101,5 @@ dbt Cloud will always connect to your data platform from the IP addresses specif
 Be sure to allow traffic from these IPs in your firewall, and include them in any database grants.
 
 Allowing these IP addresses only enables the connection to your <Term id="data-warehouse" />. However, you might want to send API requests from your restricted network to the dbt Cloud API. Using the dbt Cloud API requires allowing the `cloud.getdbt.com` subdomain. For more on the dbt Cloud architecture, see [Deployment architecture](/docs/cloud/about-cloud/architecture).
+
+[^1] Renaming Redshift, PostgreSQL, and AlloyDB connections that use an [SSH tunnel](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb#connecting-via-an-ssh-tunnel) changes the public SSH key. You must copy the new SSH key to the Bastion server, or your jobs will fail. We recommend against renaming connections in this scenario to prevent service disruptions. 
```

Comment:
```suggestion
[^1]: Renaming Redshift, PostgreSQL, and AlloyDB connections that use an [SSH tunnel](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb#connecting-via-an-ssh-tunnel) changes the public SSH key. You must copy the new SSH key to the Bastion server, or your jobs will fail. We recommend against renaming connections in this scenario to prevent service disruptions. 
```